src/hotspot/cpu/ppc/frame_ppc.hpp

```diff
@@ -118,7 +118,7 @@
     abi_reg_args_size = sizeof(abi_reg_args)
   };
 
-  #define _abi(_component) \
+  #define _abi0(_component) \
```

Comment:
When I was experimenting with this code, I thought to name it consistently with `_abi_reg_args_spill` definition a few lines below. That is, make it `_abi_reg_args`. Does that make more sense than `_abi0`?

Reply:
Sure, but lets ask @RealLucy @TheRealMDoerr first to agree on a name.

Reply:
Pardon me, I did not mean to stall this build-fixing integration. `_abi0` is an okay name, at least temporarily. We can rename it properly later.

Reply:
Please don't use `_abi_reg_args_spill` because it's often used for `abi_minframe` and other derived structures, too.
We could name it `_ppc_abi` because it refers to the layout specified in the 64-bit PowerPC ELF Application Binary Interface.
But I'm also with with `_abi0`.

Reply:
@TheRealMDoerr Why not use <_abi_reg_args>, as Aleksey suggested? Do you see an issue with that name as well?

Reply:
I think that would be confusing, because `_abi_reg_args` sounds like we had a `abi_reg_args` structure on stack which is often not the case. We are using it for any frame structure which is based on `abi_minframe`.